apps/dashboard/src/main/java/com/akto/listener/InitializerListener.java

```diff
@@ -676,6 +682,49 @@ public void accept(Account t) {
         }, 0, 15, TimeUnit.MINUTES);
     }
 
+    public void setUpSensitiveMapInApiInfoScheduler() {
+        int cronTime = 15;
+        scheduler.scheduleAtFixedRate(new Runnable() {
+            public void run() {
+                AccountTask.instance.executeTask(new Consumer<Account>() {
+                    @Override
+                    public void accept(Account t) {
+                        AccountSettings accountSettings = AccountSettingsDao.instance.findOne(AccountSettingsDao.generateFilter());
+                        CronTimers cronTimers = accountSettings.getTimers();
```

Comment:
please fix the variable naming. Crontimers, getTimers are pretty confusing

Reply:
done